1.setup.sh

```diff
@@ -64,16 +65,30 @@ idem_cmd git
 idem "command -v curl" "sudo apt install curl"
 
 function init_aws () {
+  # install CLI
   mkdir ~/awstmp
   pushd ~/awstmp
   curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
   unzip ./awscliv2.zip
   sudo ./aws/install
   popd
   rm -rf ~/awstmp
+  # defaults for CLI auth
+  mkdir ~/.aws
+  echo "[profile default]" > ~/.aws/config
+  echo "sso_start_url = $SSO_START_URL" > ~/.aws/config
+  echo "sso_region = us-west-2" > ~/.aws/config
```

Comment:
Running the three preceding lines will give you a better SSO setup.  just run `aws sso login` when your creds are expired.

Reply:
hmmm... that makes sense on a fresh install.  Given I already have an ~/.aws/config file with a [profile sand] only, should I append those three lines to the end of my existing file?